app/Http/Controllers/DashboardController.php

```diff
@@ -16,13 +19,62 @@ public function __construct()
         $this->middleware('auth');
```

Comment:
I talked to Bryn yesterday and we agreed we should stick to this format of using middleware for auth wrappers, take note @SplitSet3R/box-end team